rts/at/tx/src/main/java/org/jboss/jbossts/star/service/TMApplication.java

```diff
@@ -46,7 +46,17 @@ public class TMApplication extends Application {
     HashSet<Object> singletons = new HashSet<Object>();
     Set<Class<?>> classes = new HashSet<Class<?>> ();
 
+
     public TMApplication(Class<?> ... extraClasses) {
+        this();
+        try
+        {
```

Comment:
The style guidelines say that we should match the existing format in the file. Most of the other files implementing REST-AT don't put the opening brace on a separate line. When I wrote it they all had this style but other PRs have used a different style in places.

Reply:
Personally I prefer not to put the opening brace on a separate line, but I did so in order to match the existing format of the file (see https://github.com/jbosstm/narayana/blob/main/rts/at/tx/src/main/java/org/jboss/jbossts/star/service/TMApplication.java#L51-L52)

Reply:
The module contains a mix of positioning of braces. And REST-AT is a module containing many source files and the general style is not to put the brace on its own line. So if the second SHA contains style changes only then leave it in a separate commit.

Reply:
Could you confirm the code style commit I pushed is ok with you @mmusgrov ?